pagopa/io-app
# [IOBP-562] full state layout and basics for new payments page

## Short description
added layout and basica data gathering to said screen
DEPENDS ON https://github.com/pagopa/io-app/pull/5529
<img width="250" alt="Screenshot 2024-02-21 at 17 49 42" src="https://github.com/pagopa/io-app/assets/60693085/38ddfe98-49ba-4355-967d-a282cd87e2f7">

## List of changes proposed in this pull request
- addition of payment methods section
- addition of transaction history section
- addition of required I18n entries
- widening of PaymentCardSmall type to allow more real-world usage
- widening of PaymentCardsCarousel type to allow more real-world usage
- addition of basic data gathering an mapping for the two sections, to widen in IOBP-567

## How to test
with dev mode active, navigate to the profile and open the new payments page playground, you should be greeted by the shown screen.
